pkg/agent/openflow/client_test.go

```diff
@@ -1123,6 +1126,73 @@ func Test_client_InstallServiceGroup(t *testing.T) {
 	}
 }
 
+func getServiceBucketMaxSize(t *testing.T, ctrl *gomock.Controller) uint16 {
+	fakeOfTable := ovsoftest.NewMockTable(ctrl)
+	fakeNodeIPChecker := nodeiptest.NewFakeNodeIPChecker("2001::2")
+	ServiceLBTable.ofTable = fakeOfTable
+	defer func() {
+		ServiceLBTable.ofTable = nil
+	}()
+	fs := &featureService{
+		groupCache:    sync.Map{},
+		bridge:        binding.NewOFBridge(bridgeName, ""),
+		nodeIPChecker: fakeNodeIPChecker,
+	}
+	// To create a single-bucket group with maximum size, use a remote non-hostNetwork IPv6 Endpoint that has all
+	// available actions.
```

Comment:
I understand why you want this new test, but honestly it seems a bit hard to maintain? If we change the implementation of `serviceEndpointGroup` again, we will need to remember to come here and modify this.
Maybe if we want to test for this, the test should be more tied to the `serviceEndpointGroup` function (e.g., `TestServiceEndpointGroupMaxEdnpoints`), and we could write the test in a more functional way: create `MaxBucketsPerMessage` endpoints, call `serviceEndpointGroup` with the list of endpoints, check the size of the OF message.

Reply:
cc @wenyingd @tnqn

Reply:
Update the patch as you suggested @antoninbas